description: use the correct path to the dhcpd leases file
author: Michael Gilbert <mgilbert@debian.org>

--- a/contrib/dhcp-lease-list.pl
+++ b/contrib/dhcp-lease-list.pl
@@ -22,7 +22,7 @@ use strict;
 use warnings;
 use POSIX qw(strftime);
 
-my $LEASES = '/var/db/dhcpd.leases';
+my $LEASES = '/var/lib/dhcp/dhcpd.leases';
 my @all_leases;
 my @leases;
 
